main.go

```diff
@@ -43,9 +44,11 @@ var (
 
 func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+	utilruntime.Must(configv1.Install(scheme))
```

Comment:
curious, would like to understand better changes done to init function

Reply:
So, unless we do this the controller is unnable to read the configv1 object type. In general you end up needing to do something like this for any new CRD you want your controller to interact with.